_pkgdown.yml

```
@@ -0,0 +1,222 @@
+url: ~
+
+template:
+  bootstrap: 5
+
+reference:
+- title: Adjacency
+- contents:
+  - has_concept("adjacency")
```

Are you sure about the two dashes here and everywhere?

```suggestion
- title: Adjacency
  contents:
  - has_concept("adjacency")
```

---

yes it works and it comes straight from https://pkgdown.r-lib.org/reference/build_reference.html#reference-index (I agree it's not intuitive)